### PR TITLE
feat(fast-components-react-msft): add enzyme and unit tests for MSFT specific components

### LINKED ISSUE #731: Add enzyme for unit testing to fast-components-react-msft
Enzyme should be added and used for unit testing components that are created in the MSFT package. Additional tests are not necessary at this time when we are simply exporting a "styled" component, but components that extend or are specific to MSFT should include unit tests.

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>
closes #731 
<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-react-msft/src/flipper/flipper.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    IFlipperManagedClasses,
+    IFlipperUnhandledProps
+} from "./flipper";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("flipper snapshot", (): void => {
```
Comment (janechu) [inline packages/fast-components-react-msft/src/flipper/flipper.spec.tsx]: snapshot or snapshots? button has plural.

File: packages/fast-components-react-msft/src/button/button.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        };
+
+        const props: IButtonHandledProps & IButtonUnhandledProps = {...handledProps, ...unhandledProps};
+
+        const rendered: any = shallow(
+            <Component {...props} />
+        );
+
+        const button: any = rendered.first().shallow();
+        expect(button.prop("aria-hidden")).not.toBe(undefined);
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/src/button/button.spec.tsx]: You don't really need both of these assertions

File: packages/fast-components-react-msft/src/flipper/flipper.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+        const props: IFlipperHandledProps & IFlipperUnhandledProps = {...handledProps, ...unhandledProps};
+
+        const rendered: any = shallow(
+            <Component {...props} />
+        );
+
+        const flipper: any = rendered.first().shallow();
+
+        expect(flipper.prop("aria-labelledby")).not.toBe(undefined);
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/src/flipper/flipper.spec.tsx]: same here - the `.not.toBe(undefined` is covered with following test - the other one will fail if it is undefined

File: packages/fast-components-react-msft/src/heading/heading.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        const rendered: any = shallow(
+            <Component tag={HeadingTag.h3} />
+        );
+        const heading: any = rendered.first().shallow();
+
+        expect(heading.instance().props.tag).not.toBe(undefined);
+        expect(heading.instance().props.tag).toEqual(HeadingTag.h3);
+    });
+
+    test("should return an instance of the typography component", () => {
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/src/heading/heading.spec.tsx]: why do we care about this? If we refactor and change what this inherits from - why is that a problem as long as the prop interface is the same?

**Pull request merged**